src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/use-org-unit-cache.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+
+const useOrgUnitCache = () => {
+    const [cache, setCache] = useState(new Map())
+
+    return {
+        has: (currentRootId = null, itemId) =>
```

Comment:
The default value of `currentRootId` is `null` as in some cases the value passed to this hook will be `undefined` (as the parent component of the org unit selection components used `currentRootId={currentRoot?.id}`).